### ISSUE TITLE
Failed to load highcharter into packrat

### ISSUE THREAD

**Issue by bogdanrau** [jbkunst/highcharter#429]
On `packrat::init()`, I keep getting the following error when the local package is installed using `install_github`:
```
Error in read.dcf(...) : 
  Found continuation line starting '    shortcut functio ...' at begin of record.
```
The package installs fine locally, and also goes fine into packrat if on v 0.5 (not 0.6).

Steps to reproduce:
1. Install highcharter using `devtools::install_github("jbkunst/highcharter")`.
2. Open a new project and run `packrat::init()` with highcharter being a library that you require.
3. Notice the error.

I'm getting this on a Windows machine. I have not tested on other OS environments.

**Comment by bogdanrau** [comment]
It seems like others have had this issue in the past, mainly due to a bug in devtools that was supposedly closed (https://github.com/r-lib/devtools/issues/709). Unsure if this issue is resurfacing, if it's related to packrat, or highcharter itself. I will mention that other packages that have continuation lines do end up installing just fine, so this is really puzzling.

**Comment by jbkunst** [comment]
Hi @bogdanrau ,

Not sure how packrat works, but I think this is due there is no 0.6.0v on CRAN. Now I'm preparing the 0.6.0v release in the development (github repo) version.

**Comment by bogdanrau** [comment]
The funny part is that I have other packages from github and packrat will pick them up from wherever they are installed (github or cran). This hasn't been an issue in the past. I'll try to install from source instead of github and see if it still yells at me. I've also tried downgrading to R 3.4.3, using a newer version of devtools, using a newer version of packrat, and all have the same problem. Really puzzling.

**Comment by bogdanrau** [comment]
Here's an update. I was able to download the source code, rebuild it using `devtools::build()`, then, when running `packrat::init()`, I added the local folder where I saved the highcharter.tar.gz package at. This seemed to work, which would indicate to me that packrat should be able to install and update highcharter IF it comes from CRAN, but not GitHub. 

Again, this is actually a bug, but it may or may not come from highcharter. I think we can close this here as there's a work-around for now.
